src/main/java/org/broadinstitute/consent/http/resources/UserResource.java

```diff
@@ -333,4 +342,61 @@ public Response getSOsForInstitution(@Auth AuthUser authUser) {
             return createExceptionResponse(e);
         }
     }
+
+    @GET
+    @Produces(MediaType.APPLICATION_JSON)
+    @Path("/acknowledgements")
```

Comment:
structure-wise, would it make sense to have an `AcknowledgementResource` which has a path of `/api/user/acknowledgements`?

Reply:
My personal preference on this is to consolidate based on path so that all subjects of a path are organized in a class. 
 If you're worried about these classes getting too large, I'm happy to adjust if there are strong feelings.